src/core_plugins/i18n/server/tool/i18n_verify_keys.js

```diff
@@ -0,0 +1,87 @@
+import fs from 'fs';
```

Comment:
I'll have to give some thought to this module, but we'll need to make some changes. We need to be able to ensure that all keys have translations based not just on the files that exist, but how each plugin is configured. For example, the current tests would pass if `/src/core_plugins/kibana/i18n/en.json` exists, even if it weren't registered in the kibana plugin's init hook.